docs/modules/examples/pages/index.adoc

```diff
@@ -15,15 +15,15 @@ We're actively updating this section, so check back often!
 |===
 | Description | Colab | Documentation
 
-| Perform multi-modal RAG with LangChain, AstraDB and a Google Gemini Pro Vision model.
+| Perform multi-modal RAG with LangChain, Astra DB and a Google Gemini Pro Vision model.
```

Comment:
Should we specify Astra DB Serverless here?

Reply:
Yes.